app/uk/gov/hmrc/cataloguefrontend/serviceconfigs/ServiceConfigsController.scala

```diff
@@ -55,50 +56,113 @@ class ServiceConfigsController @Inject()(
     BasicAuthAction.async { implicit request =>
       for {
         deployments <- whatsRunningWhereService.releasesForService(serviceName).map(_.versions)
-        configByKey <- serviceconfigsService.configByKey(serviceName)
+        configByKey <- serviceConfigsService.configByKey(serviceName)
       } yield Ok(serviceConfigPage(serviceName, configByKey, deployments, appConfigBaseInSlug))
     }
 
-  def searchByKey(): Action[AnyContent] =
+  def searchByKeyLanding(): Action[AnyContent] =
+    BasicAuthAction.async { implicit request =>
+      for {
+        allTeams   <- teamsAndReposConnector.allTeams()
+        configKeys <- serviceConfigsService.configKeys()
+      } yield Ok(configSearchPage(ConfigSearch.form.fill(ConfigSearch()), configKeys, allTeams))
+    }
+
+  def searchByKeyResults(): Action[AnyContent] =
```

Comment:
I think this controller method also needs to have the form fields as queryParameters so that the url can be copy/pasted or bookmarked

similar to https://github.com/hmrc/catalogue-frontend/blob/main/app/uk/gov/hmrc/cataloguefrontend/DependencyExplorerController.scala#L82

Reply:
and then the route looks like this:

https://github.com/hmrc/catalogue-frontend/blob/main/conf/app.routes#L42

Reply:
The play form handles the query params. I'll remove the params from Dependency Explorer cause there not needed